src/views/main-view.js

```diff
@@ -64,12 +66,12 @@ MainView.prototype._initialize = function () {
     return ids;
   }.bind(this), []);
 
-  paymentMethodsViews = new PaymentMethodsView({
-    element: this.getElementById(PaymentMethodsView.ID),
+  this.paymentMethodsViews = new PaymentMethodsView({
+    element: this.element,
```

Comment:
We restructured our DOM hierarchy such that the heading label (`<div data-braintree-id="methods-label"...`) is not under the `[data-braintree-id="methods"]` element anymore. So we need to pass `this.element` here so that `PaymentMethodsView` can access the heading label.